core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -598,8 +598,19 @@ class AutoTuner(
 
   def calculateJobLevelRecommendations(): Unit = {
     val shuffleManagerVersion = appInfoProvider.getSparkVersion.get.filterNot("().".toSet)
-    appendRecommendation("spark.shuffle.manager",
-      "com.nvidia.spark.rapids.spark" + shuffleManagerVersion + ".RapidsShuffleManager")
+    val finalShuffleVersion = if (platform.contains("databricks")) {
+      val dbVersion = appInfoProvider.getProperty(
+        "spark.databricks.clusterUsageTags.sparkVersion").getOrElse("")
+      if (dbVersion.contains("10.4")) {
+        "321db"
+      } else if (dbVersion.contains("11.3")) {
+        "330db"
+      } else {
+        "332db"
+      }
+    } else shuffleManagerVersion
```

Comment:
Can we use pattern matching instead of `if-else-if`?

Reply:
I was looking into that. Can u do contains in pattern match?

Reply:
Yes, we can try something like:

```
val dbVersion = appInfoProvider.getProperty("spark.databricks.clusterUsageTags.sparkVersion").getOrElse("")
  dbVersion match {
    case version if version.contains("10.4") => "321db"
    case version if version.contains("11.3") => "330db"
    case _ => "332db"
  }
```

Reply:
sure, that seems better than what I have.

Reply:
Added.